modules/odo-getting-started-delete.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-nodejs.adoc
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-dot-net.adoc
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-go.adoc
+// * cli_reference/developer_cli_odo/getting-started-with-odo/odo-getting-started-java.adoc
+
+:_content-type: Procedure
+[id="odo-getting-started-delete_{context}"]
+
+= Deleting the resources (optional)
+
+After testing your application, you may optionally remove the resources by using the `odo delete component` command.
```

Comment:
```suggestion
After testing your application, you can remove the resources.
```
About "may," the IBM SG says: 
> Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.
> "May" is overused in technical information. In most cases, use "can" or "might" instead.

Minimalism: The command here is redundant with the one in the procedure. You can remove it.